ext/nnstreamer/tensor_filter/tensor_filter_openvino.cc

```diff
@@ -0,0 +1,424 @@
+
+failed:
+  g_critical ("Failed to get dimension information about output tensor");
+
+  return ret;
+}
+
+/**
+ * @brief The mandatory callback for GstTensorFilterFramework
+ * @param prop property of tensor_filter instance
+ * @param private_data TensorFilterOpenvino plugin's private data
+ * @param[in] input the array of input tensors
+ * @param[out] output the array of output tensors
+ * @return 0 if OK. non-zero if error
+ * @todo fill this fuction
+ */
+static int
+ov_invoke (const GstTensorFilterProperties * prop, void **private_data,
+    const GstTensorMemory * input, GstTensorMemory * output)
+{
+  return TensorFilterOpenvino::RetSuccess;
+}
+
+/**
+ * @brief The optional callback for GstTensorFilterFramework
+ * @param prop property of tensor_filter instance
+ * @param private_data TensorFilterOpenvino plugin's private data
+ * @param[out] info the dimesions and types of input tensors
+ * @return 0 (TensorFilterOpenvino::RetSuccess) if OK, negative values if error
+ */
+static int
+ov_getInputDim (const GstTensorFilterProperties * prop, void **private_data,
+    GstTensorsInfo * info)
+{
+  TensorFilterOpenvino *tfOv =
+      static_cast < TensorFilterOpenvino * >(*private_data);
+
+  g_return_val_if_fail (tfOv != nullptr, TensorFilterOpenvino::RetEInval);
+
+  return tfOv->getInputTensorDim (info);
+}
+
+/**
+ * @brief The optional callback for GstTensorFilterFramework
+ * @param prop property of tensor_filter instance
+ * @param private_data TensorFilterOpenvino plugin's private data
+ * @param[out] info The dimesions and types of output tensors
+ * @return 0 (TensorFilterOpenvino::RetSuccess) if OK, negative values if error
+ */
+static int
+ov_getOutputDim (const GstTensorFilterProperties * prop,
+    void **private_data, GstTensorsInfo * info)
+{
+  TensorFilterOpenvino *tfOv =
+      static_cast < TensorFilterOpenvino * >(*private_data);
+
+  g_return_val_if_fail (tfOv != nullptr, TensorFilterOpenvino::RetEInval);
+
+  return tfOv->getOutputTensorDim (info);
+}
+
+/**
+ * @brief Standard tensor_filter callback to open sub-plugin
+ * @return 0 (TensorFilterOpenvino::RetSuccess) if OK, negative values if error
+ */
+static int
+ov_open (const GstTensorFilterProperties * prop, void **private_data)
+{
+  std::string model_path = std::string (prop->model_files[0]);
+  std::size_t ext_start_at = model_path.find_last_of ('.');
+  std::size_t expected_len_path_model_prefix = model_path.length () - 4;
+  std::string path_model_prefix;
+  TensorFilterOpenvino *tfOv;
+
+  if ((ext_start_at == expected_len_path_model_prefix)
+      || (model_path.find (TensorFilterOpenvino::extBin,
+              ext_start_at) != std::string::npos)
+      || (model_path.find (TensorFilterOpenvino::extXml,
+              ext_start_at) != std::string::npos)) {
+    path_model_prefix = model_path.substr (0, expected_len_path_model_prefix);
+  } else {
+    path_model_prefix = model_path;
+  }
+
+  tfOv = new TensorFilterOpenvino (path_model_prefix);
```

Comment:
plz consider re-open case later.

Reply:
is this considered?

Reply:
https://github.com/nnsuite/nnstreamer/pull/1991 handles @jaeyun-jung's comment.